### ISSUE TITLE
k-9 drains battery?

### ISSUE THREAD

**Issue by mh-g** [k9mail/k-9#1290]
After the upgrade to K-9 Mail 5.010, it _seems_ that the software is draining the battery of my Samsung S5 (Cyanogenmod Marshmallow) way more than before (old K-9 version unknown). In fact, K-9 mail is by a factor of >4 largest consumer of battery power (next being Android OS).

I have four accounts configured, 3x IMAP, 1x POP3 (hourly).

**Comment by tachylyte** [comment]
I can confirm that I am having a similar issue since I updated to K-9 Mail 5.010 on my Nexus 5 running Android 6.0.1 (I have also just updated from a CM Nightly from January 2016 to a build dated 13 April 2016 so I can't  be 100% certain that it is a K-9 Mail issue).  The battery life decreases very rapidly (50% loss in about 3hrs) whenever I am not in airplane mode and K-9 Mail is reported to be the app that is using the most battery.  I (think) that I am using Push rather than a set polling interval. I have only 1 account configured.
